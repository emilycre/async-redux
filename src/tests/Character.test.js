import React from 'react';
import { shallow } from 'enzyme';
import Character from '../components/characters/Character';

describe('Character component', () => {
  it('Render a character', () => {
    const wrapper = shallow(<Character character="hello" />);
    expect(wrapper).toMatchSnapshot();
  });
});
