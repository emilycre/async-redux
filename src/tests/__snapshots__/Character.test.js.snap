// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character component Render a character 1`] = `
<section>
  <img />
  <p />
</section>
`;
